Assets/MixedRealityToolkit/_Core/Definitions/Utilities/OrientationType.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities
+{
+    /// <summary>
+    /// Orientation type enum for collections
+    /// </summary>
+    public enum OrientationType
+    {
+        /// <summary>
+        /// Don't rotate at all
+        /// </summary>
+        None,
+        /// <summary>
+        /// Rotate towards the origin
+        /// </summary>
+        FaceOrigin,
+        /// <summary>
+        /// Rotate towards the origin + 180 degrees
+        /// </summary>
+        FaceOriginReversed,
+        /// <summary>
+        /// Zero rotation
+        /// </summary>
+        FaceFoward,
+        /// <summary>
+        /// Zero rotation + 180 degrees
+        /// </summary>
+        FaceForwardReversed,
+    }
```

Comment:
Erm, it looks like this is not being ported from the latest version of the object collection. Here are very significant changes I made for the June/July release:
https://github.com/Ecnassianer/MixedRealityToolkit-Unity/commit/945401107c48493cd1e649c5a92090cf4710a7b8#diff-67828c8d64af75261de12e4344488104